docs/development.rst

```diff
@@ -505,6 +505,357 @@ point, with diagrams, and an alternative workflow, see `the git docs
+
+.. code-block:: shell
+
+    python -m maintenance add-species arabidopsis_thaliana
+
+The `maintenance` utility generates three new files inside the `catalog/{species_id}/` directory:
+
+* `__init__.py`
+* `genome_data.py`
+* `species.py`
+
+The `__init__.py` file is a wrapper script that loads all the relevant information for your species. It should be edited only when you add components to your species, such as demographic models,
+genetic maps, or DFEs.
+
+The `genome_data.py` file contains the physical map of the genome.
+The `maintenance` utility generates this file autmatically
+based on information it downloaded from Ensembl.
+Essentially, this utility sucks down a whole lot of useful information for free.
+The `genome_data.py` file contains a data dictionary which has slots for the
+assembly accession number, the assembly name,
+and a dictionary representing the chromosome names and their associated lengths.
+If synonyms are defined (e.g., chr2L for 2L) then those are given in the list that follows.
+You should double-check the downloaded values, but there is probably no reason to edit this file.
+
+The `species.py` file
+This is where you specify the mutation and recombination rates,
+effective population size, and the average generation time,
+along with all accompanying citations
+(see details in `Which information do I need to have for my species?`_).
+Inside this file are commented instructions to help you figure out where everything goes.
+Essentially, the information in this file is recorded in two main objects: `_genome` and `_species`.
+The `_genome` object contains information about the **chromosome ids** and **length**
+(taken from the  `genome_data.py` file), as well as **mutation and recombination rates**, and **ploidy**.
+
+For example, to define the `_genome` object for *A. thaliana*, we start by
+defining auxiliary objects (`_recombination_rate_data` and `_ploidy`)
+that specify the recombination rate and ploidy
+for each chromosome.
+Note that the mitochondrial and plastid genomes are associated
+with ploidy of 1 and recombination rate of 0,
+and all other chromosomes are associated with a ploidy of 2 and the
+genome-wide average recombination rate.
+
+.. code-block:: python
+
+    _mean_recombination_rate = 200 / 124000 / 2 / 1e6
+    _recombination_rate_data = {str(j): _mean_recombination_rate for j in range(1, 6)}
+    _recombination_rate_data["Mt"] = 0
+    _recombination_rate_data["Pt"] = 0
+
+    _species_ploidy = 2
+    _ploidy = {
+      "1": _species_ploidy,
+      "2": _species_ploidy,
+      "3": _species_ploidy,
+      "4": _species_ploidy,
+      "5": _species_ploidy,
+      "6": _species_ploidy,
+      "Mt": 1,
+      "Pt": 1,
+    }
+
+We then use another object (`_chromosomes`) to wrap up these
+values together with the mutation rate, which is set to the
+genome-wide average value for every chromosome:
+
+.. code-block:: python
```

Comment:
There's a nicer way to do this, which is mostly what we use now: see [here](https://github.com/popsim-consortium/stdpopsim/blob/main/stdpopsim/catalog/GasAcu/species.py#L90) for instance, using [this function](https://github.com/popsim-consortium/stdpopsim/blob/main/stdpopsim/genomes.py#L45).

Reply:
Yes, this is cleaner. I can update the example to fit this cleaner form. Probably not in the next commit that I plan to push later tonight. I assume that I should modify the actual code in AraTha to fit what I use in the doc examples, right?

Reply:
I still need to do this. I want to take care of all the other doc issues first.

Reply:
@petrelharp I rewrote the AraTha example in the docs in my latest commit. Can you check this?

Reply:
> I assume that I should modify the actual code in AraTha to fit what I use in the doc examples, right?

We can do this in a later PR (e.g. after this release) -- presumably, whomever is looking at the source code for guidance will be looking at the github head. Maybe just make an issue to point out the discrepancy.

Reply:
Looks great! (I haven't tested it runs, but looks good?)